src/datreant/core/tests/test_trees.py

```diff
@@ -234,15 +244,20 @@ def test_walk(self, tree, tmpdir):
             all_leaves = []
 
             for root, dirs, files in os.walk(tree.abspath):
+                # walk normally doesn't add slash to path, hacky fix
+                if root != tree.abspath:
```

Comment:
Please add explanation for why this is necessary; I see that it's because calling `tree.abspath` will always have the ending separator, but it might not be clear to others that read this in the future.